core/tests/protractor_desktop/embedding.js

```diff
@@ -107,28 +107,12 @@ describe('Embedding', function() {
   // These errors are to be ignored as 'idToBeReplaced' is not a valid
   // exploration id. It appears just after the page loads.
   var EMBEDDING_ERRORS_TO_IGNORE = [
-    'http:\/\/localhost:9001\/assets\/scripts\/' +
-    'embedding_tests_dev_0.0.2.min.html - Refused to display ' +
-    '\'http:\/\/localhost:9001\/embed/exploration\/idToBeReplaced',
-    'http:\/\/localhost:9001\/assets\/scripts\/' +
-    'embedding_tests_dev_0.0.2.min.html - Refused to display ' +
-    '\'http:\/\/localhost:9001\/embed\/exploration\/idToBeReplaced\\?' +
-    'locale=en#version=0.0.2&secret=',
-    'http:\/\/localhost:9001\/assets\/scripts\/' +
-    'embedding_tests_dev_0.0.2.min.html - Refused to display ' +
-    '\'http:\/\/localhost:9001\/embed\/exploration\/fake_id\\?' +
-    'locale=en#version=0.0.2&secret=',
-    'http:\/\/localhost:9001\/assets\/scripts\/' +
-    'embedding_tests_dev_i18n_0.0.1.html - Refused to display ' +
-    '\'http:\/\/localhost:9001\/explore\/idToBeReplaced',
-    'http:\/\/localhost:9001\/assets\/scripts\/' +
-    'embedding_tests_dev_i18n_0.0.1.html - Refused to display ' +
-    '\'http:\/\/localhost:9001\/explore\/idToBeReplaced\\?' +
-    'iframed=true&locale=en#version=0.0.1&secret=',
     'http:\/\/localhost:9001\/assets\/scripts\/' +
     'embedding_tests_dev_i18n_0.0.1.html - Refused to display ' +
-    '\'http:\/\/localhost:9001\/explore\/fake_id\\?' +
-    'iframed=true&locale=en#version=0.0.1&secret='
```

Comment:
Hi @kevintab95 Can you please explain me why are they deleted now. We used to get these errors in the embedding suite eariler. Thanks!

Reply:
We updated the Chrome version used for e2e and these messages changed a bit. So I just updated the messages in the test.

Reply:
Ah I see. Thanks for explaining @kevintab95! Actually there was a new error now being added in https://github.com/oppia/oppia/pull/13422 and I was confused. Thanks for explaining @kevintab95 !